src/com/group6/users/UserLogin.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Map;
+import java.util.Scanner;
+
+public class UserLogin {

```

Comment:
This class doing a bit too much. 

Consider splitting up the class as follows:
- CLI "View" by making a view say `LoginView.java` (See https://github.com/Acrylic125/sc2002-assignment/blob/main/src/com/group6/views/applicant/ApplicantHomeView.java for an example)
- Move the login validation logic to `UserManager`. The manager can have a `login(String username, String password)` method.